docs/plugins/websocket.md

```diff
@@ -109,8 +109,9 @@ export class AppComponent {
 
 Here is a list of all the available actions you have:
 
-- `ConnectWebSocket`: Action dispatched when you want to init the websocket. Optionally pass URL here.
-- `DisconnectWebSocket`: Action dispatched when the websockets disconnect.
+- `ConnectWebSocket`: Dispatch this action when you want to init the websocket. Optionally pass URL here.
+- `DisconnectWebSocket`: Dispatch this Action to disconnect a websockets.
+- `WebSocketDisconnected`: Action dispatched when websocket is disconnected. Use its reducer for reconnecting.
```

Comment:
`reducer`?

Reply:
@arturovt Would you rather say _effect_?

Reply:
@kwalski This is not NgRx :(

Reply:
- `ConnectWebSocket`: Action dispatched when you want to init the websocket. Optionally pass URL here.
- `DisconnectWebSocket`: Action dispatched when the websockets disconnect.

Please, bring these descriptions back. Also:
- `WebSocketDisconnected`: Action dispatched when the websocket is disconnected. Dispatch `ConnectWebSocket` action to init the websocket again.

Reply:
handler